build_tools/cmake/build_host_install.sh

```diff
@@ -36,5 +36,6 @@ cd build-host
 "${CMAKE_BIN?}" -G Ninja .. \
   -DCMAKE_INSTALL_PREFIX=./install \
   -DIREE_BUILD_TESTS=OFF \
-  -DIREE_BUILD_SAMPLES=OFF
+  -DIREE_BUILD_SAMPLES=OFF \
+  -DIREE_TARGET_BACKEND_CUDA=ON
```

Comment:
Any objection to enable the CUDA backend here? `build_host_install.sh` is also used in a few other tests.

Reply:
I went back and forth a bit without commenting earlier :P

It would be nice if generic scripts used default options, but I think this case is fine, since CMake now fetches what it needs for CUDA. At the point where you're using a script like this to build, having the script make a few decisions for you is pretty natural. We could also revisit making `IREE_TARGET_BACKEND_CUDA` default to `ON`, but that's a bit of a larger discussion.